backend/app/controllers/spree/admin/stores_controller.rb

```diff
@@ -114,6 +115,10 @@ def load_all_countries
         @countries = Spree::Country.all
       end
 
+      def load_all_zones
+        @zones = Spree::Zone.all
```

Comment:
```suggestion
        @zones = Spree::Zone.pluck(:id, :name)
```

Reply:
@damianlegawiec fixed, but this change also required change in form from `options_from_collection_for_select` --> `options_for_select`